packages/vuetify/package.json

```diff
@@ -100,15 +100,12 @@
     "ts-loader": "^6.2.2",
     "tslint": "^5.20.1",
     "url-loader": "^2.1.0",
-    "vue": "^3.0.1",
     "vue-meta": "^2.4.0",
-    "vue-router": "^3.0.7",
+    "vue-router": "^4.0.0-beta.13",
     "vuetify-loader": "^1.3.0",
     "webpack-dev-server": "^3.7.2"
   },
-  "peerDependencies": {
-    "vue": "^3.0.0-rc.5"
-  },
+  "peerDependencies": {},
```

Comment:
Vue needs to be in peerDependencies.

Reply:
strange. yarn removed it automatically when i removed vue from devdeps